### ISSUE TITLE
Error thrown on Xcode8 / Swift 2.3 

### ISSUE THREAD

**Issue by wspl** [ElaWorkshop/TagListView#71]
I am using CocoaPods as package manager.
TagListVeiw version is 1.0.1

Error Info:

```
file:///Users/Plutonist/Documents/Project/Rider/Rider/Base.lproj/FeedView.storyboard: error: IB Designables: Failed to render and update auto layout status for FeedViewController (9pv-A4-QxB): dlopen(TagListView.framework, 1): Symbol not found: __TWPSis16ForwardIndexTypes
  Referenced from: TagListView.framework
  Expected in: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Overlays/../../../../../../Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator/libswiftCore.dylib
 in TagListView.framework
```

```
file:///Users/Plutonist/Documents/Project/Rider/Rider/Base.lproj/FeedView.storyboard: error: IB Designables: Failed to update auto layout status: dlopen(TagListView.framework, 1): Symbol not found: __TWPSis16ForwardIndexTypes
  Referenced from: TagListView.framework
  Expected in: /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Xcode/Overlays/../../../../../../Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator/libswiftCore.dylib
 in TagListView.framework
```

The project can be compile and running normally. But this error causing the chaotic storyboard component.
What's happened and how could I solve it? Thanks!

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/37929858-error-thrown-on-xcode8-swift-2-3?utm_campaign=plugin&utm_content=tracker%2F16419772&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F16419772&utm_medium=issues&utm_source=github).

**Comment by qky1412** [comment]
same here

**Comment by anti-transmission** [comment]
Fixed by 0e25d9f.
